### ISSUE TITLE
Narzo 20 ofp unpack error

### ISSUE THREAD

**Issue by abujads** [italorecife/OppoRealme-OFP-Flash#9]
When i start program with cmd , give problem like 
Struct_error:Unpack requires buffer of 4bytes

**Comment by abujads** [comment]
Yes i have 64 bit pc , When i try with realme 5i flash file it worked , but
not work for realme narzo 20 ofp file

**Comment by jaskor** [comment]
Same issue shows up for me.  I tried on 2 computers on fedora and windows, all 64 bit and it all stops at "Struct_error:Unpack requires buffer of 4bytes". I try to flash Realme 6s global firmware from [this](https://realmefirmware.com/realme-6s-firmware/) site. Thanks for any help.

**Comment by italorecife** [comment]
sorry for the delay, so I don't know if you managed to solve it, please answer me.

**Comment by NanashiTheNameless** [comment]
im having the same issue and am also using a 64bit pc

**Comment by rexsllemel** [comment]
> sorry for the delay, so I don't know if you managed to solve it, please answer me.

I have same problem. I guess nothing to do with 32bit PC? I am using 64bit windows 10
